src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -115,6 +117,23 @@ class WorkflowManagerActor(dataAccess: DataAccess, backend: Backend) extends Act
     }
   }
 
+  /* Return value here is a tuple: (workflow name, call name) */
+  private def assertCallFqnWellFormed(callFqn: FullyQualifiedName): Future[(String, String)] = {
+    callFqn.split("\\.").toSeq match {
+      case s:Seq[String] if s.size >= 2 => Future.successful(s.head, s.last)
```

Comment:
I didn't notice until now that you were the one making it a `Seq` so same advice as above. If you explicitly make it a `List` (that's what it practically is anyways) you can use more idiomatic matching constructs